doc/oidc.md

```diff
@@ -0,0 +1,84 @@
+[<- Back to Index](index.md)
+
+# OpenID Connect Support
+
+SQL LRS supports [OpenID Connect (OIDC)](https://openid.net/connect/) on top of [OAuth 2.0](https://oauth.net/2/) for authentication and authorization to access xAPI resources and administrative functions. OIDC support enables several integration use cases:
+
+* Send xAPI statements to the LRS from OIDC-authenticated client applications
+* Provision LRS credentials and admin users programatically via the API
+* Log in to the LRS Admin UI with a foreign identity (SSO)
+
+### Resource Server
+
+SQL LRS supports OIDC token authentication to all [endpoints](endpoints.md), allowing the use of an OIDC access token to make requests. In this context SQL LRS acts as an OAuth 2.0 "resource server".
+
+To enable OIDC auth, set the `LRSQL_OIDC_ISSUER` (`oidcIssuer` in json) configuration variable to your identity provider's [Issuer Identifier](https://openid.net/specs/openid-connect-core-1_0.html#IssuerIdentifier) URI. This address must be accessible to the LRS on startup as it will perform [OIDC Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html) to retrieve public keys and other information about the OIDC environment. It is also *strongly* recommended that you set the optional `LRSQL_OIDC_AUDIENCE` (`oidcAudience`) variable to the origin address of the LRS itself (ex. "http://0.0.0.0:8080") to enable verification that a given token was issued specifically for the LRS.
```

Comment:
Shouldn't "json" be capitalized?

Reply:
https://github.com/yetanalytics/lrsql/pull/214/commits/cb2a0041a75c35ed7f68c342b5398c25c4ec657a